src/sparse_merkle_tree/in_memory_smt.rs

```diff
@@ -16,6 +16,7 @@ use crate::types::{ArcMutex, Cache, KVPair, KeyLength, NestedVec};
 type SharedInMemorySMT = JsArcMutex<InMemorySMT>;
 type DatabaseParameters = (ArcMutex<InMemorySMT>, Vec<u8>, Root<JsFunction>);
 type VerifyParameters = (Vec<u8>, NestedVec, Proof, KeyLength, Root<JsFunction>);
+type RemovedParameters = (Proof, NestedVec, Root<JsFunction>);
```

Comment:
`RemovedParameters` is quite ambiguous name. Maybe rename it to `RemovedKeysParameters`?